.github/workflows/build.yml

```diff
@@ -55,6 +55,32 @@ jobs:
         run: gradle test
         working-directory: ./java
 
+      # Upload fs-storage JNI libs
+      - name: Install cargo-ndk
+        run: cargo install cargo-ndk
+
+      - name: Add Rust Toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+          targets: aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
+
+      - name: Build fs-storage JNI libs
+        run: cargo ndk -o ./target/release/fs-storage/jniLibs --target aarch64-linux-android --target armv7-linux-androideabi --target i686-linux-android --target x86_64-linux-android build -p fs-storage --release
+
+      - name: Upload fs-storage JNI libs
+        uses: actions/upload-artifact@v4
+        with:
+          name: fs-storage-jni-libs
+          path: ./target/release/fs-storage/jniLibs
```

Comment:
Do we need to release artifacts with every commit to `main`? I'm not sure.

Reply:
It seems like running the workflow with every commit is the current preference. I had suggested a more targeted approach based on specific project updates, but received feedback that continuous execution might be better.  I'm curious to hear everyone's thoughts on finding the most optimal solution.

Reply:
Just uploading artifacts on every commit to `main` is fine and convenient for development, because we can take the lib and put it manually in ark-android. Artifacts should not be treated as releases, which we don't have set up in this repo. Artifacts expire after 3 month.

We need to setup proper releases as well, usually such are triggered by tagging a commit.